streamparse/storm/bolt.py

```diff
@@ -446,32 +403,44 @@ def _batch_entry_run(self):
                 # later batch raises an exception
                 self._batches[key] = []
             self._batches = defaultdict(list)
+            self._tick_counter = 0
 
-    def _batch_entry(self):
-        """Entry point for the batcher thread."""
-        try:
-            while True:
-                self._batch_entry_run()
-        except Exception as e:
-            log_msg = ("Exception in {}.run() while processing tuple batch "
-                       "{!r}.".format(self.__class__.__name__,
-                                      self._current_tups))
-            log.error(log_msg, exc_info=True)
-            self.raise_exception(e, self._current_tups)
+    def process(self, tup):
```

Comment:
Having this named `process` seems likely to cause confusion.  Given that `process` is what one implements in regular, it's a pretty natural mistake to then implement it in the batching bolt.  That's fine if it doesn't do anything, but it seems like it would have undefined behavior here.  Making this `_process` or `_add_to_batch` might be more descriptive and also avoid confusion for users.